### PR TITLE
feat (vendors: cookieMaxAgeSeconds) adds UI to display cookieMaxAgeSeconds in vendor stack

### PR DESCRIPTION
**Pull request by potench**
## Background
Implements disclosures for cookieMaxAgeSeconds based on this amendment: https://iabeurope.eu/wp-content/uploads/2020/08/TCF_V-CMP_comms_AmendmentsToTCFv2.0PoliciesAndTechnicalSpecificationsToRequireDisclosureOfStorageDurationPlanet49-ruling_190820_IABEurope-converted.pdf 

- [x] Updates to GVL 79
- [x] Improve legibility on Purposes List legal disclaimers ("\n" was not inserting a new line)
- [x] Implement cookieMaxAgeSeconds Device Storage disclosers in Vendors List  
- [x] Refactor error logger so we don't need to manage onerror overrides.

## Test Plan
```
yarn run
# browse to http://localhost:8080/tcf-2.0.html
# click on "Our Partners" 
# click on a vendor and scroll down to the Device Storage disclosure
```
Screen Shot 2021-02-26 at 9 15 39 AM

Screen Shot 2021-02-26 at 9 03 02 AM

Error Logger Verification:
Screen Shot 2021-02-26 at 9 32 11 AM

### CODE REVIEW

**Review by imartinez8 — APPROVED** [review]
LGTM except I think indentation might be off? Seems like really big indentations

**Comment by potench** [comment]
> LGTM except I think indentation might be off? Seems like really big indentations

Yeah - there's a really big render block with lots of ternaries and nested elements. Was thinking it's about time to break it up.

File: src/s1/tcf-2.0-cmp.js [hunk 1]
```diff
@@ -132,18 +132,16 @@ export const setup = (configOpt) => {
...
-			message: `${message} ${file && `(file: ${file})`}`,
+			message: event.message,
 		});
-		if (onErrorCache) {
-			onErrorCache(message, file, line);
-		}
-		global.onerror = onErrorCache;
+		global.removeEventListener('error', errorLoggerListener, false);
 	};
+	global.addEventListener('error', errorLoggerListener, false);
```
Comment (benstrahan) [inline src/s1/tcf-2.0-cmp.js:143]: :+1:

**Review by benstrahan — APPROVED** [review]
LGTM.

**Review by shane-landry — APPROVED** [review]

**Pull request merged**